src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1780,6 +1781,28 @@ public ProcessWindowStyle WindowStyle
         public SwitchParameter Wait { get; set; }
 
         /// <summary>
+        /// If specified, wait for this number of seconds
+        /// </summary>
+        [Parameter]
+        [Alias("TimeoutSec")]
+        [ValidateNotNullOrEmpty]
+        [ValidateRange(1, 32767)]
+        public int Timeout
```

Comment:
I think that `Timeout` is a bit misleading as it's not really a timeout on starting the process, but on how long it's expected to run.  Perhaps calling this `-WaitForExitTimeout` would be more accurate?  cc @HemantMahawar @joeyaiello

Reply:
Agreed; `-ExitTimeout` makes more sense. Would changing `-Wait` to an int cause any problems with existing scripts?

Reply:
We should probably leave `-Wait` as-is for compat reasons.  `-ExitTimeout` works for me.